### ISSUE TITLE
Publication search for academic journal articles and/or HRA lay summaries

### ISSUE THREAD

**Issue by benmeg** [opentrials/opentrials#492]
Currently the advanced search option 'Publications'...'Trials with publications' appears to return both trials with linked academic journal articles and HRA summaries.

It would be useful to separate this out, so that a user looking just for trials with academic publications is not also given trials with just HRA lay summaries (or visa-versa - a user looking just for trials with HRA lay summaries, but not linked academic journal articles - admittedly a less plausible use case, but the point is to separate the two).

**Comment by vitorbaptista** [comment]
@benmeg Maybe we could simply change the filter to return only publications as in journal articles, ignoring data from the HRA?

P.S.: We're considering HRA summaries as publications, but plan on separating them on #242

**Comment by benmeg** [comment]
@vitorbaptista Yes, good idea - returning just linked academic journal articles would be a useful improvement

**Comment by benmeg** [comment]
Once this is done, modify the 'Publications' tooltip (#315) to remove the second sentence i.e. "At present this also includes trials with lay summaries available from the UK Health Research Authority (HRA)."
